src/module-3/online-store/index.html

```diff
@@ -1,16 +1,524 @@
+                <label for="metra" class="montserrat-light-heavy-metal-16px category__item">
+                          <span class="category__item_info">
+                            <span>Metra</span>
+                            <span>55</span>
+                          </span>
+                </label>
+              </div>
+              <div>
+                <input type="radio" id="dyne" name="radio-brand" class="category__radio-btn">
+                <label for="dyne" class="montserrat-light-heavy-metal-16px category__item">
+                          <span class="category__item_info">
+                            <span>Dyne</span>
+                            <span>120</span>
+                          </span>
+                </label>
+              </div>
+              <div>
+                <input type="radio" id="lg" name="radio-brand" class="category__radio-btn">
+                <label for="lg" class="montserrat-light-heavy-metal-16px category__item">
+                          <span class="category__item_info">
+                            <span>LG</span>
+                            <span>98</span>
+                          </span>
+                </label>
+              </div>
+              <div>
+                <input type="radio" id="canon" name="radio-brand" class="category__radio-btn">
+                <label for="canon" class="montserrat-light-heavy-metal-16px category__item">
+                          <span class="category__item_info">
+                            <span>Canon</span>
+                            <span>99</span>
+                          </span>
+                </label>
+              </div>
+              <div>
+                <input type="radio" id="speck" name="radio-brand" class="category__radio-btn">
+                <label for="speck" class="montserrat-light-heavy-metal-16px category__item">
+                          <span class="category__item_info">
+                            <span>Speck</span>
+                            <span>575</span>
+                          </span>
+                </label>
+              </div>
+            </div>
+          </div>
+        </div>
+        <button class="filters__btn">
+          <span class="filters__btn_txt">CLEAR ALL FILTERS</span>
+        </button>
+      </aside>
+      <div class="flex-col-grid">
+        <div class="category">
+          <div class="montserrat-normal-heavy-metal-18px" style="align-self: center">7,618 results found</div>
+          <button class="btn-heart"><span><img class="heart-1" src="img/heart.svg" />
+              </span></button>
+        </div>
+        <div class="search-bar">
+          <input class="search-bar__input" type="search" placeholder="Search">
+          <button class="search-bar__btn" type="submit">
+            <img class="search-bar__ico" src="img/search.svg">
+          </button>
+        </div>
+        <section class="products">
+          <div class="products__item">
```

Comment:
```suggestion
          <div class="product">
```  
so then you can use `product__image`, `product__description`, `product__name` inside